### ISSUE TITLE
`pagestyle: empty` does not remove the page number on the first page

### ISSUE THREAD

**Issue by allefeld** [jgm/pandoc#6206]
Specifying `pagestyle: empty` does not remove the page number on the first page in LaTeX, if a title block is generated.
See https://tex.stackexchange.com/a/464904/38399.

That is due to this feature (bug?) of LaTeX:
https://tex.stackexchange.com/a/44281/38399

I therefore propose the workaround to include the following code in the default LaTeX template, directly after the code for `\maketitle`:

    $if(pagestyle)$
    \thispagestyle{$pagestyle$}
    $endif$

**Comment by jgm** [comment]
This seems like something that should be handled by the documentclass.
What documentclass are you using?
`book` should omit a number on the title page by default.
`article` doesn't create a title page; the title is at the beginning of the first page.

**Comment by jgm** [comment]
Also, I can imagine cases where someone would want to override the default page style but NOT affect the title -- e.g. when using the standard `book` class, which already uses a different style for the title page.

**Comment by a-vrma** [comment]
Although there doesn't seem to be a solution that works for both `article` and `book`, I will say that it is somewhat surprising to set `pagestyle` to empty and get only the first page numbered. Perhaps this could be documented?

**Comment by jgm** [comment]
I did some experimenting with adding a `titlepagestyle` variable after `\maketitle`, on the theory that we sometimes want to be able to make the title page style different.
I tried with documentclass `article`, and this worked as expected.
But with `book`, it did not!  The title page was fine, but now (despite `-Vpagestyle=empty`) we get page numbers on the first page of every chapter.
So, I don't think the proposed solution here is complete.  The problem extends beyond `\maketitle`.

**Comment by jgm** [comment]
I wonder whether a better solution would be to use fancyhdr and redefine the plain style:
In header-includes,
```
\usepackage{fancyhdr}
\fancypagestyle{plain}{%
  \fancyhf{}% clear all header and footer fields
}
```

**Comment by lemzwerg** [comment]
Yes, redefining the 'plain' pagestyle in the preamble is the way to go IMHO.
